README.md

```diff
@@ -123,7 +109,7 @@ fetch('<issuer>/.well-known/trust-token', {
 ```
 
 
-If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token attached in the Sec-Trust-Token request header and the resulting Signed Redemption Record (SRR) being expected in the Sec-Trust-Token response header.
+If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token and associated redemption metadata attached in the Sec-Trust-Token request header and the resulting Signed Redemption Record (SRR) being expected in the Sec-Trust-Token response header.
```

Comment:
What associated redemption metadata?

Reply:
The "client data" (browser public key hash and top-level origin).

Reply:
The top-level origin is coming in through the referrer, not part of the Sec-Trust-Token header. For the browser public key hash, might be worth adding it to the Trust-Bound Keypair extension down below, but otherwise for default Trust Token we don't send anything explicitly.

Reply:
OK, how about rewording to mention that extensions can potentially add related metadata beyond just the token?

Reply:
(I updated the wording to do that.)

Reply:
Marking as resolved since it seems like this is acceptable.